common/tier4_planning_rviz_plugin/include/pose_stamped_with_uuid/display.hpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef POSE_STAMPED_WITH_UUID__DISPLAY_HPP_
+#define POSE_STAMPED_WITH_UUID__DISPLAY_HPP_
+
+#include <rviz_common/message_filter_display.hpp>
+
+#include <autoware_planning_msgs/msg/pose_stamped_with_uuid.hpp>
+
+#include <deque>
+#include <memory>
+#include <string>
+
+namespace rviz_rendering
+{
+class Axes;
+class MovableText;
+}  // namespace rviz_rendering
+namespace rviz_common::properties
+{
+class FloatProperty;
+class TfFrameProperty;
+}  // namespace rviz_common::properties
```

Comment:
Any reason to declare this instead of includig `#include <rviz_common/properties/float_property.hpp>` in the header file?

Reply:
I just copy rviz_default_plugins style
https://github.com/ros2/rviz/blob/rolling/rviz_default_plugins/include/rviz_default_plugins/displays/pose_covariance/pose_with_covariance_display.hpp#L54
This may shorten the compilation by reducing includes.

Reply:
Thank you. This may reduce the readability and maintainability, but I got it.